src/main/java/org/minions/devfund/miguelcalderon/CountCharacters.java

```diff
@@ -0,0 +1,37 @@
+package org.minions.devfund.miguelcalderon;
+
+
+import java.util.logging.Logger;
+
+/**
+ * Solution for "count the characters" kata.
+ *
+ * @version 1.0 18 Jul 2018  * @author
+ * Miguel Calderon
+ */
+
+public class CountCharacters {
+    private final static Logger LOGGER = Logger.getLogger(CountCharacters.class.getName());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)